From 2e3bb1cd307105a723520b62914d5e84b3374194 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@lkc.com>
Date: Fri, 28 Mar 2014 11:50:14 -0400
Subject: [PATCH 01/10] xapp_xinit: fix mcookie path

The config scripts were including the absolute path to mcookie on the build
system rather than what the absolute path would look like on the target.
This patch hardcodes the path to the buildroot install location of mcookie
which should be good enough for running quick tests and debugging with
xinit.
---
 package/x11r7/xapp_xinit/xapp_xinit.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/x11r7/xapp_xinit/xapp_xinit.mk b/package/x11r7/xapp_xinit/xapp_xinit.mk
index da167ec..23d6e3f 100644
--- a/package/x11r7/xapp_xinit/xapp_xinit.mk
+++ b/package/x11r7/xapp_xinit/xapp_xinit.mk
@@ -8,5 +8,6 @@ XAPP_XINIT_VERSION = 1.3.2
 XAPP_XINIT_SOURCE = xinit-$(XAPP_XINIT_VERSION).tar.bz2
 XAPP_XINIT_SITE = http://xorg.freedesktop.org/releases/individual/app
 XAPP_XINIT_DEPENDENCIES = xapp_xauth xlib_libX11
+XAPP_XINIT_CONF_ENV = ac_cv_path_MCOOKIE=/usr/bin/mcookie
 
 $(eval $(autotools-package))
-- 
1.8.3.2

